mappings/net/minecraft/DispenserBehavior.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_2410 net/minecraft/DispenserBehavior
```

Comment:
Could you please move this class to the `net/minecraft/block/dispenser/` package and rename it in the mapping accordingly? This is the package `DispenserBehaviorInterface` is stored in. Enigma should be able to handle moving the class to its respective package if you write it in the name text input field.

Reply:
No problem, I think my latest commit should resolve it.